src/main/scala/fs2/aws/kinesis/RecordProcessor.scala

```diff
@@ -0,0 +1,46 @@
+package fs2
+package aws
+package kinesis
+
+import com.amazonaws.services.kinesis.clientlibrary.interfaces._
+import com.amazonaws.services.kinesis.clientlibrary.types._
+import com.amazonaws.services.kinesis.clientlibrary.lib.worker.ShutdownReason
+import scala.collection.JavaConverters._
+
+
+private[kinesis] class RecordProcessor(cb: CommittableRecord => Unit) extends v2.IRecordProcessor {
+   private var shardId: String = _
+   private var extendedSequenceNumber: ExtendedSequenceNumber = _
+   var shutdown: Option[ShutdownReason] = None
+   var latestCheckpointer: Option[IRecordProcessorCheckpointer] = None
```

Comment:
I think there's different spacing here, at least seems so on Github.

Reply:
or it's the def's underneath,  not sure which is right.

Reply:
fixed